docs/src/surfaces/card/card.jsx

```diff
@@ -95,20 +94,19 @@ export default class CardOnClickSample extends React.Component {
 
     render() {
         return (
-            <Card compact={true} nest={true} onClick={this._onClick.bind(this)}>
-                <Grid stressed={true}>
-                    <Grid.Row>
-                        <Grid.Column style={{ padding: 0, width: '44px' }}>
-                            <img style={{ display: 'block' }} src="http://placehold.it/44x44/dbe0e3/1c2530" />
-                        </Grid.Column>
-
-                        <Grid.Column style={{ padding: '0 0 0 22px', width: 'auto' }}>
-                            <Header size="small" sub={true}>
-                                A Title
-                                <Header.Subheader>Lorem ipsum dolor sit amet, consectetur adipiscing.</Header.Subheader>
-                            </Header>
-                        </Grid.Column>
-                    </Grid.Row>
+            <Card compact={true} nest onClick={this._onClick.bind(this)}>
+                <Grid spacing={2}>
+                    <Grid.Column style={{ padding: 0, width: '44px' }}>
+                        <img style={{ display: 'block' }} src="http://placehold.it/44x44/dbe0e3/1c2530" />
+                    </Grid.Column>
+
+                    <Grid.Column style={{ padding: '0 0 0 22px', width: 'auto' }}>
+                        <Typography size="small" sub={true}>
+                            A Title
+                            </Typography>
```

Comment:
Weird indentation here.  Also, I don't think `<Typography>` needs `sub` prop, right?  That's a vestige of `<Header>`?  Also `size` prop as well (not needed/appropriate for `<Typography>`)?

Same issues in a few places throughout here.

Reply:
Fixed.

Reply:
Weird indentation is fixed.  Other issues not fixed.  But they can be NABs.

Reply:
I believe I reverted this one too. Let's let these pass.

Reply:
Sounds good.  Resolving the thread.  We can do other cleanups later.